### PR TITLE
Wizard documentation draft.

### PR DESCRIPTION
**Pull request by DianaKorladinova**
Signed-off-by: DianaKorladinova <chanchunche@gmail.com>

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [x] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Linked to # 1370 in zowe/api-layer
A brief description of the YAML Wizard and a short guide on how to use it.
:heart:Thank you!

### CODE REVIEW

File: docs/extend/extend-apiml/onboard-wizard.md [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+
+As an API developer, you can use the Yaml Onboarding Wizard to simplify the process of onboarding new REST API services to the Zowe API Mediation Layer. The wizard offers a walkthrough of the required steps to create a correct configuration file which is used to set the application properties and Eureka metadata.
+
 ## Onboarding your REST service with the Wizard
-1. Click on the 'Onboard New API' dropdown located in the navbar  
+
+The following procedure describes how to onboard your REST service with the Wizard.
+
+**Follow these steps:**
+
```
Comment (janan07) [inline docs/extend/extend-apiml/onboard-wizard.md:10]: I thought this article was going to include how to install the wizard...

File: docs/extend/extend-apiml/onboard-wizard.md [hunk 2]
```diff
@@ -1,12 +1,18 @@
...
+1. In the ... , click the 'Onboard New API' dropdown located in the navbar.
```
Comment (janan07) [inline docs/extend/extend-apiml/onboard-wizard.md:11]: Where is the user in this first step?

**Review by janan07 — COMMENTED** [review]
Initial Review by Tech Info

File: docs/extend/extend-apiml/onboard-wizard.md (hunk 1 above)
Comment (ViktorKorladinov) [reply docs/extend/extend-apiml/onboard-wizard.md:10]: The wizard is embedded in the webpage. The user would just need to click the dropdown, and the wizard will pop up as a modal dialog.

File: docs/extend/extend-apiml/onboard-wizard.md (hunk 2 above)
Comment (ViktorKorladinov) [reply docs/extend/extend-apiml/onboard-wizard.md:11]: Main page, the button is in the navbar. We will add that to the doc.

File: docs/extend/extend-apiml/onboard-wizard.md (hunk 1 above)
Comment (janan07) [reply docs/extend/extend-apiml/onboard-wizard.md:10]: Ok. What webpage?

File: docs/extend/extend-apiml/onboard-wizard.md (hunk 2 above)
Comment (janan07) [reply docs/extend/extend-apiml/onboard-wizard.md:11]: Main page?

File: docs/extend/extend-apiml/onboard-wizard.md (hunk 1 above)
Comment (ViktorKorladinov) [reply docs/extend/extend-apiml/onboard-wizard.md:10]: It's part of the API catalog. We are working on localhost so we don't know, but should we find out the link to the deployed version and paste it there?

File: docs/extend/extend-apiml/onboard-wizard.md (hunk 2 above)
Comment (ViktorKorladinov) [reply docs/extend/extend-apiml/onboard-wizard.md:11]: Main page of the API catalog. The first thing the user sees when he logs in - the dashboard.

File: docs/extend/extend-apiml/onboard-wizard.md (hunk 1 above)
Comment (janan07) [reply docs/extend/extend-apiml/onboard-wizard.md:10]: Yes. Let's be specific when we have the deployed version.

**Comment by janan07** [comment]
Reviewed and edited by Tech Info

**Pull request merged**